dotman-plugin/src/main/java/net/minevn/dotman/commands/AdminCmd.kt

```diff
@@ -260,5 +261,28 @@ class AdminCmd {
                 }}
             }
         }
+
+        private fun traCuuMaGD() = command {
```

Comment:
Tên function nên viết bằng tiếng Anh:
```suggestion
        private fun getTransaction() = command {
```
hoặc nếu viết bằng tiếng Việt:
```suggestion
        private fun traCuuGiaoDich() = command {
```